packages/react-components/icons/CheckCircle.tsx

```diff
@@ -9,8 +9,8 @@ interface Props {
 
 export default class CheckCircle extends React.PureComponent<Props> {
   static defaultProps = {
```

Comment:
This class is called `CheckCircle` but it's not a check anymore :P Also, there's a radio button that looks just like this one called `RadioIcon` on `src/icons/RadioIcon.tsx` maybe we can leave just one of these?

Reply:
hahaha :P i was gonna name it "RadioButton" oh dang - I'll delete one of them!